modules/models/src/main/java/org/eclipse/xpanse/modules/models/service/DeployResult.java

```diff
@@ -45,7 +45,11 @@ public class DeployResult {
      * The result property of the service task.
      */
     @NotNull
-    @Schema(description = "The result property of the service task.")
-    private Map<String, String> property = new HashMap<>();
+    @Schema(description = "The properties of the deployed service.")
+    private Map<String, String> deployedServiceProperties = new HashMap<>();
```

Comment:
we can keep the field property as it is and renaming the `deployResultProperties` to `rawProperties`, may be properties is better?